.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,46 @@
+name: Python Code Quality
```

Comment:
If these workflows fail, I suppose it is easy for the developer to reproduce the failure locally (using the instructions you have added recently); correct?

Reply:
Yeah so basically this workflow here automates enforcing that the developer actually ran the python code quality tools using our instructions on the Pull Request Template. i.e. reproducing entails just running the checks locally which is already documented e.g. for formatting: 

```bash
$ black .
```